.azure-pipelines/run_hpu_tests.py

```diff
@@ -0,0 +1,151 @@
+"""This file is called from the hpu-tests.yml pipeline.
+
+The following script run the hpu tests in parallel.
+Tests run are:
+1. test_inference_only is run on four cards
+2. test_all_stages on two cards
+3. complete hpu tests using one card
+4. complete hpu tests using eight cards.
+"""
+import itertools
+import subprocess
+import sys
+
+HPU_TESTS_DICTIONARY = {
```

Comment:
It will skipped. As we will be running just this python file

```bash
    - bash: |
         python ".azure-pipelines/run_hpu_tests.py"
      displayName: 'HPU Tests in parallel'
 ```

Reply:
that is my point, pls stay with standard `pytest` search